buildHelpers/createRollupConfig.js

```diff
@@ -0,0 +1,103 @@
+import typescript from '@rollup/plugin-typescript';
+import { terser } from 'rollup-plugin-terser';
+
+// type RollupOptions = {
+//   format: 'umd' | 'cjs' | 'esm',
```

Comment:
It feels weird that we accept a format, but still need to call separate functions. Could we remove the export of `createUMDRollupConfig` and `createCJS_ESMRollupConfig` and export a single `createRollupConfig` that calls the appropriate function based on the format option?

Reply:
I was also thinking it didn't have to be too extreme of a refactor, just 
```
export function createUMDRollupConfig(options) {
  if(options.format === 'umd') {
    return createUMDRollupConfig(options);
  } else if(['cjs', 'esm].includes(options.format)) {
    return createCJS_ESMRollupConfig(options);
  } else {
    throw new Error("unknown format");
  }
}